[
   {
      "author_association" : "NONE",
      "body" : "I currently have:\r\n```\r\n19:07:21 Ã¯Â¿Â¼ getblockhash 477893 \r\n19:07:47 Ã¯Â¿Â¼ 00000000000000000131d93cad98313b8c64ad61220b704c62dd9ef454972403\r\n```\r\nBut, am running that node pruned so don't know if I have that block or if the hash was fetched.",
      "created_at" : "2018-01-26T08:10:09Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-360711777",
      "id" : 360711777,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-01-26T08:15:46Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/360711777",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/1173622?v=4",
         "events_url" : "https://api.github.com/users/Willtech/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Willtech/followers",
         "following_url" : "https://api.github.com/users/Willtech/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Willtech/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Willtech",
         "id" : 1173622,
         "login" : "Willtech",
         "organizations_url" : "https://api.github.com/users/Willtech/orgs",
         "received_events_url" : "https://api.github.com/users/Willtech/received_events",
         "repos_url" : "https://api.github.com/users/Willtech/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Willtech/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Willtech/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Willtech"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "Dam really ok look yea I no ok and I've knew all along duh boo!!!\r\n\r\nmark.a.wortham\r\n\r\nOn Jan 26, 2018 2:11 AM, Willtech <notifications@github.com> wrote:\r\n\r\nI currently have:\r\n\r\n19:07:21 Ã¯Â¿Â¼ getblockhash 477893\r\n19:07:47 Ã¯Â¿Â¼ 00000000000000000131d93cad98313b8c64ad61220b704c62dd9ef454972403\r\n\r\n\r\nÃ¢ÂÂ\r\nYou are receiving this because you are subscribed to this thread.\r\nReply to this email directly, view it on GitHub<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbitcoin%2Fbitcoin%2Fissues%2F12256%23issuecomment-360711777&data=02%7C01%7C%7Cc10665df52cb4812c82008d5649466eb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636525510899766174&sdata=d827tYCLnEkz8YDwjDv0WuV4EAYNepoAMoJkXkvr4KU%3D&reserved=0>, or mute the thread<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAcUokcE-lRjkp1glLyxB3gxxwbmPUHsmks5tOYisgaJpZM4RqwoW&data=02%7C01%7C%7Cc10665df52cb4812c82008d5649466eb%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636525510899766174&sdata=Pr%2BD4xcBKAwRk3tYP94%2BO2kRWJnsrleNXg7lEQA75Zk%3D&reserved=0>.\r\n\r\n",
      "created_at" : "2018-01-26T08:25:42Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-360714809",
      "id" : 360714809,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-01-26T08:25:42Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/360714809",
      "user" : {
         "avatar_url" : "https://avatars3.githubusercontent.com/u/29698193?v=4",
         "events_url" : "https://api.github.com/users/markaw67/events{/privacy}",
         "followers_url" : "https://api.github.com/users/markaw67/followers",
         "following_url" : "https://api.github.com/users/markaw67/following{/other_user}",
         "gists_url" : "https://api.github.com/users/markaw67/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/markaw67",
         "id" : 29698193,
         "login" : "markaw67",
         "organizations_url" : "https://api.github.com/users/markaw67/orgs",
         "received_events_url" : "https://api.github.com/users/markaw67/received_events",
         "repos_url" : "https://api.github.com/users/markaw67/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/markaw67/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/markaw67/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/markaw67"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "I am now having a similar problem with 0.14.2 occurring earlier in the blockchain. I have a node running on another machine that is synced to the network. on my unix box that I'm trying to get up I edited the bitcoin.conf file to connect directly to the full node I had running. Next I wrote a script to keep the sync going:\r\n`#!/bin/bash\r\n# bin directory for bitcoind and bitcoin-cli\r\nlaunchdir=$HOME/Downloads/bitcoin-0.14.2/bin\r\n# launch the bitcoin daemon and count the number of errors in the debug log\r\nsudo $launchdir/bitcoind -datadir=/var/lib/bitcoind -conf=/etc/bitcoin/bitcoin.conf &\r\ncounterold=$(grep -o 'ERROR' /var/log/bitcoind/debug.log | wc -l)\r\n# wait for a minute to allow the daemon to sync with the network\r\nsleep 60\r\n# initialize the block height\r\nBLOCK=$($launchdir/bitcoin-cli getblockcount)\r\n# set the period to run until approximately synched with the network\r\nwhile [ $BLOCK -lt 506095 ]\r\ndo\r\n        # count the number of errors in the log file and determine the block height\r\n        counter=$(grep -o 'ERROR' /var/log/bitcoind/debug.log | wc -l)\r\n        BLOCK=$($launchdir/bitcoin-cli getblockcount)\r\n        # skip the block reconsideration if there is no change in error numbers in the log file\r\n        while [ $counterold -lt $counter ]\r\n        do\r\n                # report the location of the hung block and reconsider that block\r\n                echo Block synchronization is stuck on block height $BLOCK\r\n                HASH=$($launchdir/bitcoin-cli getblockhash $BLOCK)\r\n                $launchdir/bitcoin-cli reconsiderblock $HASH\r\n                counterold=$counter\r\n        done\r\n        # wait 2 seconds\r\n        sleep 2\r\ndone\r\n# congrats you are done\r\necho Successfully synced to the network!`\r\n\r\nMy current block height is 468188 and is slowly progressing. The error counter in the debug.log file is at 6327 and rising.",
      "created_at" : "2018-01-26T12:25:34Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-360771851",
      "id" : 360771851,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-01-26T12:25:34Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/360771851",
      "user" : {
         "avatar_url" : "https://avatars1.githubusercontent.com/u/9408922?v=4",
         "events_url" : "https://api.github.com/users/crabel99/events{/privacy}",
         "followers_url" : "https://api.github.com/users/crabel99/followers",
         "following_url" : "https://api.github.com/users/crabel99/following{/other_user}",
         "gists_url" : "https://api.github.com/users/crabel99/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/crabel99",
         "id" : 9408922,
         "login" : "crabel99",
         "organizations_url" : "https://api.github.com/users/crabel99/orgs",
         "received_events_url" : "https://api.github.com/users/crabel99/received_events",
         "repos_url" : "https://api.github.com/users/crabel99/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/crabel99/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/crabel99/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/crabel99"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "Could it be that this is a hardware issue with that machine?",
      "created_at" : "2018-01-27T12:18:44Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-360981137",
      "id" : 360981137,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-01-27T12:18:44Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/360981137",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/1173622?v=4",
         "events_url" : "https://api.github.com/users/Willtech/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Willtech/followers",
         "following_url" : "https://api.github.com/users/Willtech/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Willtech/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Willtech",
         "id" : 1173622,
         "login" : "Willtech",
         "organizations_url" : "https://api.github.com/users/Willtech/orgs",
         "received_events_url" : "https://api.github.com/users/Willtech/received_events",
         "repos_url" : "https://api.github.com/users/Willtech/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Willtech/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Willtech/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Willtech"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "I don't think so. I ran bitcoind on this computer under Mac OS from 2012-2017. It chugged along the entire time without issue. I used to even mine with it back in the day... The blocks sync a few at a time when I force it to reconsider the block in the above script. There has got to be a configuration that is off that is causing the syncing difficulty. I have the stock bitcoin.conf file with a few updates for RPC and connecting to trusted peers, but other than that it runs on defaults. On my other MacOS and Windows 10 boxes, the defaults are adequate to sync with the network.",
      "created_at" : "2018-01-27T16:04:11Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-360994474",
      "id" : 360994474,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-01-27T16:04:11Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/360994474",
      "user" : {
         "avatar_url" : "https://avatars1.githubusercontent.com/u/9408922?v=4",
         "events_url" : "https://api.github.com/users/crabel99/events{/privacy}",
         "followers_url" : "https://api.github.com/users/crabel99/followers",
         "following_url" : "https://api.github.com/users/crabel99/following{/other_user}",
         "gists_url" : "https://api.github.com/users/crabel99/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/crabel99",
         "id" : 9408922,
         "login" : "crabel99",
         "organizations_url" : "https://api.github.com/users/crabel99/orgs",
         "received_events_url" : "https://api.github.com/users/crabel99/received_events",
         "repos_url" : "https://api.github.com/users/crabel99/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/crabel99/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/crabel99/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/crabel99"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "Using the script above I was able to force the client to sync to the network. Once I was synced to the network, I shifted `bitcoin.conf` to the default (moving my known node from `connect` to `addnode`). I shifted the path from 0.14.2 to 0.15.99 and the daemon converted the downloaded chain to the newer version and with a few more `bitcoin-cli reconsiderblock` commands it finally started relaying transactions and keeping the chain up to date.\r\n\r\nI am not sure why this was so difficult. Some things to observe. Because my hardware was able to sync to the network and is now properly relaying transactions, it is not a hardware issue. There is some other problem that I have no idea what it is. There are still a considerable amount of connection errors. \r\n\r\n> 2018-01-28 15:55:29 New outbound peer connected: version: 70015, blocks=506512, peer=96\r\n> 2018-01-28 15:55:33 Misbehaving: 35.185.46.13:8333 peer=96 (0 -> 100) BAN THRESHOLD EXCEEDED\r\n> 2018-01-28 15:55:37 New outbound peer connected: version: 70015, blocks=506412, peer=94\r\n> 2018-01-28 15:55:42 New outbound peer connected: version: 70015, blocks=506512, peer=97\r\n> 2018-01-28 15:55:54 connect() to 91.205.47.233:8333 failed after select(): No route to host (113)\r\n> 2018-01-28 15:55:56 New outbound peer connected: version: 70013, blocks=224331, peer=98\r\n> 2018-01-28 15:56:00 version handshake timeout from 93\r\n> 2018-01-28 15:56:07 Misbehaving: 171.8.83.19:8333 peer=79 (60 -> 80)\r\n> 2018-01-28 15:56:07 ERROR: non-continuous headers sequence\r\n> 2018-01-28 15:56:19 New outbound peer connected: version: 70015, blocks=506512, peer=99\r\n> 2018-01-28 15:56:35 Misbehaving: 5.9.101.190:8333 peer=97 (0 -> 100) BAN THRESHOLD EXCEEDED\r\n> 2018-01-28 15:56:36 Misbehaving: 185.145.130.92:8333 peer=99 (0 -> 100) BAN THRESHOLD EXCEEDED\r\n> 2018-01-28 15:56:44 connect() to 66.172.112.181:8333 failed after select(): Connection refused (111)\r\n> 2018-01-28 15:56:46 Misbehaving: 10.1.10.139:8333 peer=0 (21700 -> 21800)\r\n> 2018-01-28 15:56:46 Misbehaving: 10.1.10.139:8333 peer=0 (21800 -> 21900)\r\n> 2018-01-28 15:56:46 Misbehaving: 10.1.10.139:8333 peer=0 (21900 -> 22000)\r\n> 2018-01-28 15:56:46 Misbehaving: 10.1.10.139:8333 peer=0 (22000 -> 22100)\r\n> 2018-01-28 15:56:46 Misbehaving: 10.1.10.139:8333 peer=0 (22100 -> 22200)\r\n> 2018-01-28 15:56:46 Misbehaving: 10.1.10.139:8333 peer=0 (22200 -> 22300)\r\n> 2018-01-28 15:56:47 New outbound peer connected: version: 70015, blocks=506512, peer=101\r\n> 2018-01-28 15:56:50 connect() to [2604:9a00:2010:a08d:10::23]:8333 failed after select(): Connection refused (111)\r\n> 2018-01-28 15:56:51 connect() to 91.214.82.68:8333 failed after select(): Connection refused (111)\r\n> 2018-01-28 15:56:52 Misbehaving: 120.79.83.179:8333 peer=101 (0 -> 100) BAN THRESHOLD EXCEEDED\r\n> 2018-01-28 15:56:52 Misbehaving: 10.1.10.139:8333 peer=0 (22300 -> 22400)\r\n> 2018-01-28 15:56:58 connect() to 51.15.7.224:8333 failed after select(): Connection refused (111)",
      "created_at" : "2018-01-28T16:01:20Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-361073436",
      "id" : 361073436,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-01-28T16:01:20Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/361073436",
      "user" : {
         "avatar_url" : "https://avatars1.githubusercontent.com/u/9408922?v=4",
         "events_url" : "https://api.github.com/users/crabel99/events{/privacy}",
         "followers_url" : "https://api.github.com/users/crabel99/followers",
         "following_url" : "https://api.github.com/users/crabel99/following{/other_user}",
         "gists_url" : "https://api.github.com/users/crabel99/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/crabel99",
         "id" : 9408922,
         "login" : "crabel99",
         "organizations_url" : "https://api.github.com/users/crabel99/orgs",
         "received_events_url" : "https://api.github.com/users/crabel99/received_events",
         "repos_url" : "https://api.github.com/users/crabel99/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/crabel99/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/crabel99/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/crabel99"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "You have some serious errors noted in your log there, you even have:\r\n`2018-01-28 15:56:07 ERROR: non-continuous headers sequence` and `2018-01-28 15:56:46 Misbehaving: 10.1.10.139:8333 peer=0 (21700 -> 21800)` - I presume this is your local peer that is ban exempt because it was specified?\r\n\r\nI do not know what the issue is but I will leave further attempts to diagnose to those who have coded the software. There is certainly some issue between your peers.",
      "created_at" : "2018-01-30T08:32:24Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-361513326",
      "id" : 361513326,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-01-30T08:37:11Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/361513326",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/1173622?v=4",
         "events_url" : "https://api.github.com/users/Willtech/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Willtech/followers",
         "following_url" : "https://api.github.com/users/Willtech/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Willtech/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Willtech",
         "id" : 1173622,
         "login" : "Willtech",
         "organizations_url" : "https://api.github.com/users/Willtech/orgs",
         "received_events_url" : "https://api.github.com/users/Willtech/received_events",
         "repos_url" : "https://api.github.com/users/Willtech/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Willtech/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Willtech/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Willtech"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "The non-continuous headers sequence message is not a serious error, its an uninteresting report that you get when some altcoin nodes connect to you.",
      "created_at" : "2018-02-04T09:12:50Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-362892340",
      "id" : 362892340,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-02-04T09:12:50Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/362892340",
      "user" : {
         "avatar_url" : "https://avatars1.githubusercontent.com/u/858454?v=4",
         "events_url" : "https://api.github.com/users/gmaxwell/events{/privacy}",
         "followers_url" : "https://api.github.com/users/gmaxwell/followers",
         "following_url" : "https://api.github.com/users/gmaxwell/following{/other_user}",
         "gists_url" : "https://api.github.com/users/gmaxwell/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/gmaxwell",
         "id" : 858454,
         "login" : "gmaxwell",
         "organizations_url" : "https://api.github.com/users/gmaxwell/orgs",
         "received_events_url" : "https://api.github.com/users/gmaxwell/received_events",
         "repos_url" : "https://api.github.com/users/gmaxwell/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/gmaxwell/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/gmaxwell/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/gmaxwell"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "@gmaxwell So by presumption, the OP has some issue with the local node?",
      "created_at" : "2018-02-04T10:00:00Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-362895118",
      "id" : 362895118,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-02-04T10:00:00Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/362895118",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/1173622?v=4",
         "events_url" : "https://api.github.com/users/Willtech/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Willtech/followers",
         "following_url" : "https://api.github.com/users/Willtech/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Willtech/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Willtech",
         "id" : 1173622,
         "login" : "Willtech",
         "organizations_url" : "https://api.github.com/users/Willtech/orgs",
         "received_events_url" : "https://api.github.com/users/Willtech/received_events",
         "repos_url" : "https://api.github.com/users/Willtech/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Willtech/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Willtech/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Willtech"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "You might want to check your hardware for defects just to be safe.\r\n* memtest86 to check your RAM\r\n* to check the CPU behaviour under load, use linpack or Prime95\r\n* to test your storage device use smartctl or CrystalDiskInfo\r\n\r\nSource: https://bitcoin.stackexchange.com/a/12206",
      "created_at" : "2018-02-04T18:20:19Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/12256#issuecomment-362927516",
      "id" : 362927516,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12256",
      "updated_at" : "2018-02-04T18:20:19Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/362927516",
      "user" : {
         "avatar_url" : "https://avatars0.githubusercontent.com/u/6399679?v=4",
         "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
         "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
         "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
         "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/MarcoFalke",
         "id" : 6399679,
         "login" : "MarcoFalke",
         "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
         "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
         "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/MarcoFalke"
      }
   }
]
